README.md

```diff
@@ -19,3 +19,21 @@ Creates PDF with Quarkus extensions contain given dependency (quarkus-vertx, qua
 - Usage:`get-extensions.sh DEPENDENCY_NAME [OUTPUT_FILE_NAME].pdf` 
 - Example: `./get-extensions.sh quarkus-vertx extensions-vertx.pdf`
 - Result will be written by default to the PDF document `Quarkus-extensions-with-*.pdf` or to the file with your custom name
+
+### artifact-version-diff
+
+Create the report of different version between two Qaurkus version. This can be used for upstream vs RHBQ or upstream vs upstream.
+
+- Preparation:
+  - `git clone https://github.com/quarkusio/quarkus.git`
+  - `git checkout <tag>` tag should be version which will be base for comparison e.g. 3.8.3
+  - `mvn versions:compare-dependencies -DremotePom=com.redhat.quarkus.platform:quarkus-bom:<version_to_compare> -Dmaven.repo.local="<path_to_local_repository>" -DreportOutputFile=<name_of_output_file>`
```

Comment:
Btw, it would also be useful to describe what `remotePom` and `maven.repo.local` should contain?

Or maybe use different property names altogether?

Thinking about it more, maybe it would be the best to be able altogether specify what platform boms we want to compare? This part is a bit unclear to me upon re-review.

Reply:
> Btw, it would also be useful to describe what remotePom and maven.repo.local should contain?

I can describe these more 

> Or maybe use different property names altogether?

As versons is mojo plugin I can't change them.

> As in, the workflow I would prefer is to specify 2 GAVs for platform BOMs to compare and let the program output the table for me instead of having to do prep manually.

When creating this I look at this posibility but didn't find any solution or hint how to do this. Only other solutions which I was able to come up was comparing the boms directly (we can't know if the extension don't have own version of artifact, found two places). Second solution would be have some generic pom containing the extensions and create dependency tree.

I choose using versions plugin as it can show more then other solutions which I come up.

Reply:
I don't understand now at all - what is being compared against what when we add these parameters?

Maybe the plugin should be executed from inside the application itself, what do you think?

Reply:
We comparing the version defined in Quarkus repository vs some pom/bom

`remotePom` is the bom against which the upstream is compared (e.g. `com.redhat.quarkus.platform:quarkus-bom:3.8.3.redhat-00001`)
`upstream` is quarkus repository checkouted to tag which will be base for comparison  (e.g. `3.8.3`)
`reportOutputFile` is just output file which will contain the different artifacts. It's generated for each module in quarkus pom.

> Maybe the plugin should be executed from inside the application itself, what do you think?

I'll look at this possibility and try to add it

Reply:
@jedla97 maybe a shell script for starters, instead?

One that would just take Quarkus product version + Maven repository zip, infer upstream Quarkus tag from it and launch the 

```
mvn clean install exec:java -DskipTests -Dmaven.repo.local=${MAVEN_REPO_LOCAL} -Dquarkus.platform.bom=com.redhat.quarkus.platform:quarkus-bom:${QUARKUS_PLATFORM_BOM_VERSION} -Dquarkus.repo.tag=${QUARKUS_TAG}
```

A starting point, at least.